### PR TITLE
updated container activity event class.

### PR DESCRIPTION
**Pull request by ashishvermarox**
Updated container activity event class.
Added UID and some name changes.
Tested the json files on local server, which resulted in 0 errors.
Screenshot of UI attached.
Screen Shot 2022-08-11 at 4 27 38 PM

 Signed-off-by: Ashish Verma [ashish_verma@live.com](mailto:ashish_verma@live.com)

### CODE REVIEW

File: events/container/container_activity.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+	"caption": "Base Container Activity",
+	"category": "container",
+	"description": "This is a base class for the container activity classes.",
+	"extends": "container_lifecycle",
+	"name": "container_activity",
+	"uid": 0,
```
Comment (rroupski) [inline events/container/container_activity.json:7]: If this class is meant to be the base class for the other container classes, then I'd suggest to remove the uid to hide the class from the schema browser.

File: events/container/lifecycle.json [hunk 2]
```diff
@@ -1,25 +1,24 @@
...
-          "caption": "Start"
-        },
-        "4": {
-          "caption": "Stop"
-        }
-      }
-    }
-  }
+	"description": "Container lifecycle  events report installation, removal, start, and stop of containers.",
+	"extends": "base_event",
```
Comment (rroupski) [inline events/container/lifecycle.json:3]: Perhaps you want to "extends": "container_activity" to get all attributes defined in the container_activity class?

File: events/container/lifecycle.json [hunk 3]
```diff
@@ -1,25 +1,24 @@
...
-        "3": {
-          "caption": "Start"
-        },
-        "4": {
-          "caption": "Stop"
-        }
-      }
-    }
-  }
+	"description": "Container lifecycle  events report installation, removal, start, and stop of containers.",
```
Comment (rroupski) [inline events/container/lifecycle.json:2]: You will need to add `"uid": 1` to make the class visible on the schema browser.

**Review by rroupski — CHANGES_REQUESTED** [review]

**Comment by ashishvermarox** [comment]
Hello @rroupski, 
Below is summary of changes as requested.
1. container.json is removed.
2. container_activity is base class, with no UID.
3. lifecycle.json is extending conatiner_activity, with UID 1, so that it is visible on browser.

**Review by rroupski — APPROVED** [review]

**Review by rroupski — APPROVED** [review]

**Pull request merged**